interface-elements-for-web/articles/grid/filtering/filtering-grid-data.md

```diff
@@ -18,13 +18,12 @@ If the dropdown does not display check boxes, click the required value to apply
 
 ![ASpxGridView_HeaderFilter](../../../images/img7159.png)
 
-If filtering is applied, the filter dropdown only displays values that match the filter criteria. To remove the filter, click **(All)**.
+The filter dropdown only displays values that match the applied filter criteria. To remove the filter, click **(All)**.
 
 Note that if a filter is applied to a column, other column header filters display unique values of the sorted rows. To show the full list of values (include values of rows hidden by sorting), hold down SHIFT and click a header filter button.
```

Comment:
Note that if a filter is applied to a column, other column header filters display the sorted rows' unique values. You can press SHIFT and click a header filter button to show all the values (including hidden rows' values).